policyengine_us/variables/gov/states/la/tax/income/credits/school_readiness/la_school_readiness_credit_non_refundable.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class la_school_readiness_credit_non_refundable(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana non-refundable school readiness tax credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.revenue.louisiana.gov/IndividualIncomeTax/SchoolReadinessTaxCredit"
+    defined_for = StateCode.LA
+
```

Comment:
Could we replace the formula with this?
```suggestion
    adds = ["la_school_readiness_tax_credit"]
    subtracts = ["la_school_readiness_tax_credit_refundable"]
```

Reply:
@XZ-mabel yes we can

Reply:
@PavelMakarchuk Do we need to keep la_child_tax_credit_non_refundable.yaml for testing la_school_readiness_credit_non_refundable.py after I made this change?